quantum/rgblight_list.h

```diff
@@ -13,111 +13,163 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-#ifndef RGBLIGHT_LIST_H
-#define RGBLIGHT_LIST_H
+#pragma once
+
+/*              RGB COLORS             */
+#define RGB_WHITE        0xFF, 0xFF, 0xFF
+#define RGB_RED          0xFF, 0x00, 0x00
+#define RGB_CORAL        0xFF, 0x7C, 0x4D
+#define RGB_ORANGE       0xFF, 0x80, 0x00
+#define RGB_GOLDENROD    0xD9, 0xA5, 0x21
+#define RGB_GOLD         0xFF, 0xD9, 0x00
+#define RGB_YELLOW       0xFF, 0xFF, 0x00
+#define RGB_CHARTREUSE   0x80, 0xFF, 0x00
+#define RGB_GREEN        0x00, 0xFF, 0x00
+#define RGB_SPRINGGREEN  0x00, 0xFF, 0x80
+#define RGB_TURQUOISE    0x47, 0x6E, 0x6A
+#define RGB_TEAL         0x00, 0x80, 0x80
+#define RGB_CYAN         0x00, 0xFF, 0xFF
+#define RGB_AZURE        0x99, 0xf5, 0xFF
+#define RGB_BLUE         0x00, 0x00, 0xFF
+#define RGB_PURPLE       0x7A, 0x00, 0xFF
+#define RGB_MAGENTA      0xFF, 0x00, 0xFF
+#define RGB_PINK         0xFF, 0x80, 0xBF
+
+/*            HSV COLORS            */
+#define HSV_WHITE          0,   0, 255
```

Comment:
Could you add a `HSV_MATRIX_*` section here, as well? 

Specifically, the rgblight feature uses Hue properly (0-360) while rgb matrix uses incorrectly (0-255).  So using this for rgb matrix may be close, but it's not exact.